mautic/mautic
# Small grammar fix

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | [ ] N/A
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A

#### Description:

While importing some contacts testing another PR I realised this message was using incorrect grammar:

> Finish on the background

It should say 

> Finish in the background

This PR addresses that small change!

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Import contacts (a large number - use mockaroo.com to generate some data)
3. Notice the message changes to be correct grammar.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10977"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
